### PR TITLE
Add nb_black to environment yml

### PR DESCRIPTION
**Pull request by MarcoGorelli**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -5,3 +5,4 @@ channels:
 dependencies:
   - python=3
   - folium
+  - nb_black=1.0.5=py_0
```
Comment (simonjayhawkins) [inline environment.yml]: @MarcoGorelli do we need to pin the version?

Comment (MarcoGorelli) [reply environment.yml]: I guess not

Comment (MarcoGorelli) [reply environment.yml]: (nobody expects the version number!)

**Comment by jakirkham** [comment]
Thanks @MarcoGorelli and @simonjayhawkins!

**Pull request merged**